db/schema.rb

```diff
@@ -10,17 +10,31 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20171024183246) do
+ActiveRecord::Schema.define(version: 20171222165527) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "books", force: :cascade do |t|
     t.string "title"
+    t.bigint "reviews_id"
```

Comment:
Keep in mind that the foreign key goes on the resource that has only one of the other.  So, there is a direct correlation from a model having a belongs_to and a model have a foreign key on it. 

In this scenario I would want to see a user_id and book_id on reviews, but not a review_id on books. 

Also note that a review belongs to one user and one book. Your foreign keys will always be singular since you are linking to one other specific record.